tests/foreman/cli/test_classparameters.py

```diff
@@ -0,0 +1,827 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Smart/Puppet Class Parameter"""
+
+from robottelo.decorators import run_only_on, stubbed, tier1, tier3
+from robottelo.test import CLITestCase
+
+
+class SmartClassParametersTestCase(CLITestCase):
+    """Implements Smart Class Parameter tests in CLI"""
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_override_check(self):
+        """Override the Default Parameter value.
+
+        @feature: Class Parameters - Override
+
+        @steps:
+
+        1.  Override the parameter.
+        2.  Set the new valid Default Value.
+        3.  Submit the changes.
```

Comment:
See my answer on your questions 'Steps 1 and 2 mean the same thing here right?'.